engine/src/main/java/org/hibernate/validator/internal/metadata/aggregated/ContainerCascadingMetaData.java

```diff
@@ -77,21 +76,13 @@
 	 */
 	private final boolean hasContainerElementsMarkedForCascading;
 
-	/**
-	 * The set of value extractors which are type compliant and container element compliant with the element where
-	 * the cascaded validation was declared. The final value extractor is chosen among these ones, based on the
-	 * runtime type.
-	 */
-	private final Set<ValueExtractorDescriptor> valueExtractorCandidates;
-
```

Comment:
You can also remove `getValueExtractorCandidatesForCascadedValidation()` from `ValueExtractorManager` and the 2 private methods it calls.